release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3322,4 +3322,28 @@ $ oc adm release info 4.11.29 --pullspecs
 [id="ocp-4-11-29-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-30"]
+=== RHSA-2023:1030 - {product-title} 4.11.30 bug fix and security update
+
+Issued: 2023-03-07
+
+{product-title} release 4.11.30, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:1030[RHSA-2023:1030] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:1029[RHBA-2023:1029] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.30 --pullspecs
+----
+
+[id="ocp-4-11-30-bug-fixes"]
+==== Bug fixes
+* Previously, when creating a `Secret`, the *Start Pipeline* model created an invalid JSON value, As a result, the `Secret` was unusable and the `PipelineRun` could fail. With this fix, the *Start Pipeline* model creates a valid JSON value for the Secret. Now, you can create valid Secrets while starting a Pipeline. (link:https://issues.redhat.com/browse/OCPBUGS-7494[OCPBUGS-7494*])
```

Comment:
```suggestion
* Previously, when creating a `Secret`, the *Start Pipeline* model created an invalid JSON value. As a result, the `Secret` was unusable and the `PipelineRun` could fail. With this fix, the *Start Pipeline* model creates a valid JSON value for the `Secret`. Now, you can create valid secrets while starting a pipeline. (link:https://issues.redhat.com/browse/OCPBUGS-7494[OCPBUGS-7494*])
```

Reply:
Last sentence: I don't think we normally capitalize these when just referring to them as general objects (and not menu items/system objects).

Reply:
Is the asterisk supposed to be there at the end of the Jira ticket number?